ngsi_adapter/src/test/acceptance/Vagrantfile

```diff
@@ -0,0 +1,45 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-WARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+
+#__author__ = 'jfernandez'
+
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  # Every Vagrant virtual environment requires a box to build off of.
+  config.vm.box = "hashicorp/precise32"
+
+  # Provision
+  config.vm.provision "shell", inline: "cd /home/vagrant", privileged: true
+  config.vm.provision "shell", inline: "wget https://bootstrap.pypa.io/get-pip.py", privileged: true
+  config.vm.provision "shell", inline: "python get-pip.py", privileged: true
+  config.vm.provision "shell", inline: "apt-get update", privileged: true
+  config.vm.provision "shell", inline: "apt-get -y install python-dev", privileged: true
+  config.vm.provision "shell", inline: "apt-get -y install git", privileged: true
+  config.vm.provision "shell", inline: "apt-get -y install libxml2-dev libxslt1-dev", privileged: true
+  config.vm.provision "shell", inline: "pip install -r /vagrant/requirements.txt", privileged: true
+end
```

Comment:
The Vagrantfile has been created to be used with the box: *[hashicorp/precise32](https://atlas.hashicorp.com/hashicorp/boxes/precise32)*, a standard Ubuntu 12.04 LTS 32-bit box, defined in the line 34 of the Vagrantfile script.